src/keybindings.h

```diff
@@ -154,6 +154,7 @@ enum GeanyKeyBindingID
 	GEANY_KEYS_FILE_CLOSE,						/**< Keybinding. */
 	GEANY_KEYS_DOCUMENT_REPLACETABS,			/**< Keybinding. */
 	GEANY_KEYS_FILE_RELOAD,						/**< Keybinding. */
+	GEANY_KEYS_FILE_RELOAD_ALL,						/**< Keybinding. */
```

Comment:
Please make additions at the end of the list or itr breaks the ABI

Reply:
Oh, sorry, didn't know so I put it next to the other reload. I see an API number and version in a comment, should I add one and increment?

Reply:
Good point, you have added to the API, so yes inc the number in `apidata.h`, and "since API ..." in the comment.

Reply:
couldn't find apidata but a plugindata.h had a number to increment

Reply:
Correct, sorry.  I did say I was short of time, I'm commenting between other things so they may contain erorrs or be incomplet.